nextflow.config

```diff
@@ -2,15 +2,17 @@ params {
     output_dir = 'results'
     upload_to_synapse = false
     synapse_folder_id = null
+    use_gpt = false
 }
 
 process {
     withName: ProcessRepo {
         container = 'bitnami/git:2.44.0'
     }
-    
-    withName: GenerateReport {
-        container = 'ubuntu:22.04'
+
+    withName: 'AIAnalysis' {
+        container = 'ghcr.io/sage-bionetworks/synapsepythonclient:latest'
+        debug = true
```

Comment:
You don't need to have container set here if it is set in the process definition. Also is debug still meant to be turned on?